FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/Models/FhirToDataLakeAzureStorageJobInfo.cs

```diff
@@ -0,0 +1,51 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using Microsoft.Health.Fhir.Synapse.JobManagement.Extensions;
+using Microsoft.Health.Fhir.Synapse.JobManagement.Models;
+using Newtonsoft.Json;
+
+namespace Microsoft.Health.Fhir.Synapse.Core.Jobs.Models
+{
+    public class FhirToDataLakeAzureStorageJobInfo : AzureStorageJobInfo
+    {
+        private readonly DateTimeOffset _fakeDataEndTime = new DateTimeOffset(2022,1,1,0,0,0, TimeSpan.Zero);
```

Comment:
Can we use DateTimeOffset.Min/Max to  avoid some magic numbers?

Reply:
Sure, updated.